src/mono/msbuild/android/build/AndroidBuild.props

```diff
@@ -6,16 +6,16 @@
 
     <_IsLibraryMode Condition="'$(NativeLib)' != ''">true</_IsLibraryMode>
 
-    <AndroidBuildAppAfterThisTarget Condition="'$(AndroidBuildAppAfterThisTarget)' == ''">Publish</AndroidBuildAppAfterThisTarget>
-    <AndroidBuildAppDependsOn>
+    <AndroidBuildAfterThisTarget Condition="'$(AndroidBuildAfterThisTarget)' == ''">Publish</AndroidBuildAfterThisTarget>
+    <AndroidBuildDependsOn>
         _InitializeCommonProperties;
-        _BeforeAndroidBuildApp;
+        _BeforeAndroidBuild;
         _AndroidResolveReferences;
         _AndroidPrepareProfiledAot;
-        _AndroidAotCompileApp;
+        _AndroidAotCompile;
         _BuildNativeLibrary;
```

Comment:
I think we should make this into a property and set the one that should be used depending if its app build or library build before importing AndroidBuild.props. In library builder SDK we could always set the property to _BuildNativeLibrary.

Reply:
I'd prefer doing this in a follow up.